src/HIE/Bios/Cradle.hs

```diff
@@ -300,12 +355,18 @@ obeliskWorkDir fp = do
   unless check (fail "Not obelisk dir")
   return wdir
 
-
-obeliskCradle :: FilePath -> Cradle
-obeliskCradle wdir =
-  Cradle {
-      cradleRootDir  = wdir
-    , cradleOptsProg = CradleAction "obelisk" (obeliskAction wdir)
+obeliskCradleDependencies :: FilePath -> IO [FilePath]
+obeliskCradleDependencies _wdir = return []
```

Comment:
Couldnt find any obelisk files in an very shallow search (mainly the obelisk gh page)